cookbooks/aws-parallelcluster-environment/recipes/config/mount_cx_fs.rb

```diff
@@ -10,15 +10,34 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
+return if on_docker?
+efs_shared_dir_array = node['cluster']['efs_shared_dirs'].split(',')
+efs_fs_id_array = node['cluster']['efs_fs_ids'].split(',')
+efs_encryption_in_transit_array = node['cluster']['efs_encryption_in_transits'].split(',')
+efs_iam_authorization_array = node['cluster']['efs_iam_authorizations'].split(',')
 
-# Mount EFS directory with efs resource
+cx_shared_dir_array = []
```

Comment:
Minor: AFAIK CX stands for Customer Experience, so it may be not the right acronym.

Reply:
I've seen this a few times in tickets and other chats https://t.corp.amazon.com/D90018305/communication

Reply:
I think we can split the `mount_cx_fs` in two recipes: `efs` and `fsx`.  

In this way we'll follow the convetion used for raid and ebs and we'll address Giacomo's suggestion about using `cx` word. We don't need to have a wrapper `mount_*_fs` recipe to call `fsx` and `efs` mount, we can move them in the config.rb

Reply:
I think we can split the mount_cx_fs in two recipes: `efs` and `fsx`.

In this way we'll follow the convention used for raid and ebs and we'll address Giacomo's suggestion about using cx word. We don't need to have a wrapper `mount_*_fs` recipe to call fsx and efs mount, we can move them in the config.rb

Reply:
I use CX as many others do to refer to customers.  There are many instances where folks are using this.  I am using this to distinguish cx shared drives from the internal use shared drives.  Given that users can add both EFS and FSx filesystems, I'm not sure I see the value in separating them now, when they weren't already separated before.